### DISCUSSION TITLE
New User: I have no Tools tab.

### DISCUSSION THREAD

**Discussion by shaun-lloyd** [vlachoudis/bCNC#1752] [Q&A]
Hi, i am trying to use the drag knife plugin. I don't have a tools tab. I have /usr/share/bcnc/bCNC/plugins/dragknife.py as per ubuntu apt-get install.

I simple don't know how to "activate" / show the tools tab.

thank you.

**Comment by imechura** [comment]
Me too

**Comment by MARIOBASZ** [comment]
TOOL is renamed as CAM
